src/Pages/Profile/admin/AdminView.jsx

```diff
@@ -0,0 +1,196 @@
+      accessLevel: userMembership,
+      numberOfSemestersToSignUpFor: numberOfSemestersToSignUpFor
+    }
+
+    setUser({ ...editedUser })
+    setToggle(!toggle)
+
+    await axios
+      // get all user!
+      .post('/api/user/edit', {
+        ...editedUser,
+        queryEmail: queryEmail,
+        token: props.token
+      })
+      .then(result => {
+        if (result.status >= 200 && result.status < 300) {
+          setMembershipValidUntil(result.data.membershipValidUntil)
+        }
+      })
+      .catch(err => {
+        console.log(err)
+      })
+
+    setToggle(false)
+    setToggleSubmit(false)
+  }
+
+  function handleSubmissionToggle() {
+    if (user.email === email)
+      setToggleSubmit(!toggleSubmit)
+    else {
+      checkIfUserExists(email)
+    }
+  }
+
+  async function checkIfUserExists(val) {
+    await axios
+      .post('/api/user/checkIfUserExists', { email: val })
+      .then(result => {
+        if (result.status >= 200 && result.status < 300) {
+          setToggleSubmit(!toggleSubmit)
+        }
+      })
+      .catch(err => {
+        window.alert('The Email has already existed, leave it blank if you do not want to change')
+        console.log(err)
+      })
+  }
+
+  const formGroups = [
+    {
+      label: 'First Name',
+      placeholder: user.firstName,
+      handleChange: (e) => setFirstName(e.target.value)
+    },
+    {
+      label: 'Last Name',
+      placeholder: user.lastName,
+      handleChange: (e) => setLastName(e.target.value)
+    },
+    {
+      label: 'Middle Initial',
+      placeholder: user.middleInitial,
+      handleChange: (e) => setMiddleInitial(e.target.value)
+    },
+    {
+      label: 'Email',
+      type: 'email',
+      placeholder: user.email,
+      handleChange: (e) => setEmail(e.target.value)
+    },
+    {
+      label: 'Password',
+      type: 'password',
+      placeholder: 'make it secure',
+      handleChange: (e) => setPassword(e.target.value)
+    },
+    {
+      label: 'Door Code',
+      placeholder: 'make it secure',
+      handleChange: (e) => setDoorCode(e.target.value)
+    },
+    {
+      label: 'Major',
+      placeholder: user.major,
+      handleChange: (e) => setMajor(e.target.value)
+    }
+  ]
+
+  const membership = [
+    { value: 0, name: 'Keep Same' },
+    { value: 0, name: 'Expired Membership' },
+    { value: 1, name: 'This semester' },
+    { value: 2, name: '2 semesters' }
+  ]
+
+  return (
+    <div className="center">
+      <ul className="profileInfo">
+
+        <Display 
+        user={user} 
+        membershipValidUntil={membershipValidUntil} 
+        />
+
+        <EditForm 
+        formGroups = {formGroups}
+        membership = {membership}
+        setNumberOfSemestersToSignUpFor ={ (e)=>{setNumberOfSemestersToSignUpFor(e)} }
+        setPagesPrinted = {e=>{setPagesPrinted(e)}}
+        handleSubmissionToggle = {()=>{handleSubmissionToggle()}}
+        handleToggle = {()=>{handleToggle()}}
```

Comment:
you can remove the `handleToggle` function entirely with

```suggestion
        handleToggle = {()=>{setToggle(!toggle)}}
```